rhel7/profiles/stig_gui.profile

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+metadata:
+    version: V3R1
```

Comment:
Update the version in the metadata too.

Reply:
It is still V1R1, :grimacing:

Reply:
hahaha my bad.. was looking at the wrong profile...whoops